doc/install/installation.md

```diff
@@ -0,0 +1,346 @@
+
+    mysql> CREATE USER 'huginn'@'localhost' IDENTIFIED BY '$password';
+
+Ensure you can use the InnoDB engine which is necessary to support long indexes
+
+    mysql> SET storage_engine=INNODB;
+
+    # If this fails, check your MySQL config files (e.g. `/etc/mysql/*.cnf`, `/etc/mysql/conf.d/*`)
+    # for the setting "innodb = off"
+
+Grant the Huginn user necessary permissions on the database
+
+    mysql> GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER, LOCK TABLES ON `huginn_production`.* TO 'huginn'@'localhost';
+
+Quit the database session
+
+    mysql> \q
+
+Try connecting to the new database with the new user
+
+    sudo -u huginn -H mysql -u huginn -p -D huginn_production
+
+    # Type the password you replaced $password with earlier
+
+You should now see `ERROR 1049 (42000): Unknown database 'huginn_production'` which is fine because we will create the database later.
+
+You are done installing the database and can go back to the rest of the installation.
+
+
+## 6. Huginn
+
+### Clone the Source
+
+    # We'll install Huginn into home directory of the user "huginn"
+    cd /home/huginn
+
+    # Clone Huginn repository
+    #sudo -u huginn -H git clone https://github.com/cantino/huginn.git -b master huginn
+    # **FIXME**
+    sudo -u huginn -H git clone https://github.com/dsander/huginn.git -b deployment-guide huginn
+
+    # Go to Huginn installation folder
+    cd /home/huginn/huginn
+
+    # Copy the example Huginn config
+    sudo -u huginn -H cp .env.example .env
+
+    # Create the log/, tmp/pids/ and tmp/sockets/ directories
+    sudo -u huginn mkdir -p log tmp/pids tmp/sockets
+
+    # Make sure Huginn can write to the log/ and tmp/ directories
+    sudo chown -R huginn log/ tmp/
+    sudo chmod -R u+rwX,go-w log/ tmp/
+
+    # Make sure permissions are set correctly
+    sudo chmod -R u+rwX,go-w log/
+    sudo chmod -R u+rwX tmp/
+    sudo -u huginn -H chmod o-rwx .env
+
+    # Copy the example Unicorn config
+    sudo -u huginn -H cp config/unicorn.rb.example config/unicorn.rb
+
+### Install Gems
+
+**Note:** As of bundler 1.5.2, you can invoke `bundle install -jN` (where `N` the number of your processor cores) and enjoy parallel gem installation with measurable difference in completion time (~60% faster). Check the number of your cores with `nproc`. For more information check this [post](http://robots.thoughtbot.com/parallel-gem-installing-using-bundler). First make sure you have bundler >= 1.5.2 (run `bundle -v`) as it addresses some [issues](https://devcenter.heroku.com/changelog-items/411) that were [fixed](https://github.com/bundler/bundler/pull/2817) in 1.5.2.
+
+    sudo -u huginn -H bundle install --deployment --without development test
+
+
+### Configure it
+
+    # Update Huginn config file and follow the instructions
+    sudo -u huginn -H editor .env
```

Comment:
It set the `HOME` env variable, not sure if its needed for all commands but it works :smile: